Task7.java

```diff
@@ -0,0 +1,10 @@
+public class Task7 {
+    public static void main(String[] args) {
+        int a = 1 + (int) (Math.random() * 99 );
+        int b = 1 + (int) (Math.random() * 99 );
+        int c = 1 + (int) (Math.random() * 99 );
```

Comment:
You don't need to convert to int, Math.random() returns double, use it instead.
Also you can put a value of random func in output like that:
System.out.println(1 + Math.random()*99);

Reply:
fixed